style.css

```diff
@@ -1,4 +1,233 @@
 * {
-  box-sizing: border-box;
-  margin: 0;
+  /*box-sizing: border-box;*/
+  margin: 0 auto;
+  position: relative;
+}
+
+html {
+  background: url(https://s-media-cache-ak0.pinimg.com/736x/05/71/c0/0571c02063ea91a8dca4c75858712609.jpg) no-repeat center center fixed;
+  -webkit-background-size: cover;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  background-size: cover;
+  position: relative;
+}
+
+body {
+  background-repeat: no-repeat;
+
+}
+.left{
+  text-align: left;
+  position: fixed;
+  bottom: 10%;
+  left: 5%;
+  width: 19rem;
+  height: 5rem;
+  transform: translate(-8rem, 0rem);
+  transform: skewY(-3deg);
+  overflow: auto;
+}
+
+ .avatar{
+  float: left;
+
+ }
+
+ .hp {
+   margin-top: 1.5rem;
+   font-family: Rockwell;
+   color: white;
+   font-size: .7rem;
+ }
+
+ .hp--500 {
+   font-size: 1.5rem;
+   font-family: Rockwell;
+ }
+
+.health {
+  border-bottom: white 2px solid;
+  border-right: #209fd5 3px solid;
+  transform: skewX(-15deg);
+  overflow: auto;
+  width: 9rem;
+  float: left;
+}
+ .bars {
+  background-color: white;
+  float: left;
+  margin-bottom: .5rem;
+  border: .01rem solid gray;
+  width: .35rem;
+  height: 1rem;
+  transform: skewX(-5deg);
+  opacity: .65;
+}
+
+.bars--max {
+  background-color:yellow;
+  float: left;
+  margin: 0rem;
+  border: .01rem solid #9c9b7e;
+  transform: skewX(-5deg);
+  width: .35rem;
+  height: 1rem;
+}
+
+.right {
+  transform: skewY(3deg);
+  position: fixed;
+  bottom: 2%;
+  right: 5%;
+  overflow: auto;
+  width: 20rem;
+
+}
+
+.weapon {
+  float: right;
+  position: relative;
+  margin-right: .5rem;
+
+
+}
+.weapon-border {
+  border-right: #209fd5 4px solid;
+  height: 2rem;
+  position: absolute;
+  left: 98%;
+  top: 36%;
+  transform: skewX(-14deg);
+  margin-left: .3rem;
+  opacity: .45;
+}
+
+.abilities {
+  top: 30%;
+  right: 50%;
+  position: fixed;
+  transform: skewX(-15deg);
+}
+
+.abilities--1 {
+  border: white 1px solid;
+  border-radius: 20%;
+  background-color: #c6c6c6;
+  opacity: .65;
+
+}
+
+.abilities--2 {
+  border: white 1px solid;
+  border-radius: 20%;
+  background-color: #c6c6c6;
+  opacity: .65;
+
+}
+
+.key1 {
+  position: absolute;
+  left: 10%;
+  opacity: .85;
+  border: #c6c6c6 1px solid;
+  padding: 0 2px;
+  color: white;
+  transform: skewX(-10deg);
+  border-radius: 20%;
+}
+
+.key2 {
+  position: absolute;
+  left: 67%;
+  opacity: .85;
+  border: #c6c6c6 1px solid;
+  padding: 0 3px;
+  color: white;
+  transform: skewX(-10deg);
+  border-radius: 20%;
+}
+.ammo {
+  color: white;
+  float: right;
+  position: absolute;
+  top: 1rem;
+  right: .5rem;
+  font-size: .7rem;
+  transform: skewX(-10deg);
+  font-family: Rockwell;
+
+}
+.ammo--41 {
+  font-size: 1.3rem;
+  font-family: Rockwell;
+}
+
+.middle {
+  position: fixed;
+  top: 62%;
+  right: 36%;
```

Comment:
If you want to center this up on the right to left axis, set this to `right: 50%` and use the `transform: translate` trick https://rtablada.gitbooks.io/summer-2016/content/week-1/day-3/transform.html